packages/elements/src/components.d.ts

```diff
@@ -2640,6 +2644,10 @@ declare namespace LocalJSX {
           * Styles the row in a selected style.  In contrast to `activated`, use this option to select one or multiple items that are likely to change soon.
          */
         "selected"?: boolean;
+        /**
+          * Specifies the tabindex of the list item.
+         */
+        "tabIndex"?: number;
```

Comment:
To prevent mirroring all HTML Attributes we could add generic attrs prop which could be spreaded to the internal HTML element.

```
  /**
   * HTML Properties
   */
  @Prop() attrs: JSXBase.HTMLAttributes<HTMLLIElement>;

[...]

  <li ref={(el) => (this.listItemEl = el)} class={listItemClasses} {...this.attrs}>
```

To make use of the `JSXBase.HTMLAttributes` typ the `tsconfig` compiler Options should be extended by 

    "skipLibCheck": true,